vroom/clangformat.vroom

```diff
@@ -180,6 +180,62 @@ that clang-format has a version >= 3.4).
 
 
 
+For clang-format versions 3.8 and above, it will sort #include lines (using the
+-sort-includes option).
+
+  :Glaive codefmt clang_format_executable='clang-format-3.7'
+  @clear
+  % #include <stdlib.h><CR>
+  |#include <stdio.h><CR>
+  |int f() { int i = 1; return 1234567890; }
+
+  :goto 1
```

Comment:
Huh, now I know of at least 4 different ways to go to the beginning of line 1: `1gg`, `1G`, `:1`, and `:goto 1`. Since `:goto 1` is byte-oriented, I might avoid it except when using it with other byte counts; line-oriented operations are more versatile in common practice.